query-languages/m/type-replacetablekeys.md

```diff
@@ -13,4 +13,66 @@ Type.ReplaceTableKeys(<b>tableType</b> as type, <b>keys</b> as list) as type
 
 ## About
 
-Returns a new table type with all keys replaced by the specified list of keys.
+Returns a new table type with all keys replaced by the specified list of keys. 
+
+Each key is defined using a record in the following form: 
+```powerquery-m
+[
```

Comment:
This key example is kind of non-standard. What does { text } represent, for example? Not saying the way we've done it is ideal, but usually we'd describe the key format, maybe with a bulleted list of fields, and then show how it works via the examples.